infertrade/utilities/operations.py

```diff
@@ -151,6 +151,28 @@ def moving_average(x: Union[np.ndarray, pd.Series], window: int) -> np.ndarray:
     return ma_np
 
 
+def weighted_moving_average(x: Union[np.ndarray, pd.Series], window: int) -> np.ndarray:
```

Comment:
Where is the new function used? We can replace duplicated code?

Reply:
@ta-oliver 
I am just going through codes in this repo for the first time. Sorry for my confusion but I don't see most of the functions in operations.py being used anywhere?
For example, where is a function like moving_average used?

Reply:
Looks like it is not being used?

https://github.com/ta-oliver/infertrade/search?q=moving_average&type=code

Reply:
Perhaps was originally planned for use here: https://github.com/ta-oliver/infertrade/blob/86a60fc438d138b06d3ccef8f878244c4c79a7e1/infertrade/algos/community/allocations.py#L119

Reply:
We could remove if not being used?